external/java/src/main/java/ch/epfl/dedis/lib/omniledger/contracts/DarcInstance.java

```diff
@@ -135,6 +135,88 @@ public void evolveDarcAndWait(Darc newDarc, Signer owner) throws CothorityExcept
         }
     }
 
+    /**
+     * Creates an instruction for spwaning a contract.
```

Comment:
s/spwaning/spawning/

Reply:
difficult word - I found another occurrence of `spwan`... but only in java...